### PR TITLE
feat(login): add main header description

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/1487

### CODE REVIEW

**Comment by mcoker** [comment]
cc @kybaker @mceledonia

File: src/patternfly/components/Login/login-main-header-desc.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<p class="pf-c-login__main-header-desc{{#if login-main-header-desc--modifier}} {{login-main-header-desc--modifier}}{{/if}}"
```
Comment (christiemolloy) [inline src/patternfly/components/Login/login-main-header-desc.hbs]: Will you add this to the docs :)

File: src/patternfly/components/Login/login.scss [hunk 2]
```diff
@@ -179,13 +184,31 @@
...
     }
   }
 }
 
+.pf-c-login__main-header-desc {
+  margin-bottom: var(--pf-c-login__main-header-desc--MarginBottom);
+  font-size: var(--pf-c-login__main-header-desc--FontSize);
+  grid-column: 1 / -1;
+
+  @media (max-width: $pf-global--breakpoint--md) {
```
Comment (christiemolloy) [inline src/patternfly/components/Login/login.scss]: can you explain why you didn't add the media query at the top where you define the variables?

Comment (mcoker) [reply src/patternfly/components/Login/login.scss]: that's an oversight, updated.

**Comment by mcoker** [comment]
@christiemolloy updated with those changes.

File: src/patternfly/components/Login/docs/code.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
 | `.pf-c-login__header .pf-c-brand` | `<img>` |  Creates a brand image inside of login header. |
 | `.pf-c-login__main` | `<main>` |  Positions the login main area. |
 | `.pf-c-login__main-header` | `<header>` |  Creates the header of the main area. |
+| `.pf-c-login__main-header-desc` | `<p>` |  Creates the description in the main area header. |
```
Comment (christiemolloy) [inline src/patternfly/components/Login/docs/code.md]: another question can you have `.pf-c-login__main-header-desc` without `.pf-c-login__main-header` and will that look ok? If not maybe add required to header?

Comment (mcoker) [reply src/patternfly/components/Login/docs/code.md]: created https://github.com/patternfly/patternfly-next/issues/1572 to track that.

**Pull request merged**